hypertrace-gradle-publish-maven-central-plugin/src/main/java/org/hypertrace/gradle/publishing/PublishMavenCentralPlugin.java

```diff
@@ -73,18 +81,22 @@ private void configureNexusStagingPlugin() {
   }
 
   private void applyMavenPublish() {
-    project.getPluginManager()
-      .apply(MavenPublishPlugin.class);
+    project.getPluginManager().apply(MavenPublishPlugin.class);
```

Comment:
Everything beyond here should just be the application of our standard formatter.